pkg/blobplugin/Dockerfile

```diff
@@ -28,7 +28,15 @@ RUN chmod +x /blobfuse-proxy/init.sh && \
  chmod +x /blobfuse-proxy/blobfuse-proxy.service && \
  chmod +x /blobfuse-proxy/blobfuse-proxy
 
-RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase
+RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates uuid-dev util-linux mount udev wget e2fsprogs nfs-common netbase curl
+RUN if [ "$ARCH" == "amd64" ] ; then \
+  curl -sL https://aka.ms/downloadazcopy-v10-linux | tar -xz && \
```

Comment:
there is also https://aka.ms/downloadazcopy-v10-linux-amd64, so you could append `$ARCH` to https://aka.ms/downloadazcopy-v10-linux-$ARCH, that would make Dockerfile logic quite simpler

Reply:
This link seems not be useful. https://aka.ms/downloadazcopy-v10-linux-amd64

Reply:
then use a var, e.g.
```
azcopyURL = https://aka.ms/downloadazcopy-v10-linux
if arm64 then
  azcopyURL = azcopyURL + "-arm64"
fi

curl -sL azcopyURL | tar -xz && \
mv ./azcopy_linux_$ARCH_*/azcopy /usr/local/bin/azcopy && \
rm -rf ./azcopy_linux_$ARCH_*; fi

```